### PR TITLE
Add base backup script for moving standalone to HA

### PR DESCRIPTION
**Pull request by heheh13**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ TAG        ?= $(shell git describe --exact-match --abbrev=0 2>/dev/null || echo
 
 DOCKER_PLATFORMS := linux/amd64 linux/arm64
 PLATFORM         ?= linux/$(subst x86_64,amd64,$(subst aarch64,arm64,$(shell uname -m)))
-VERSION          = $(TAG)_$(subst /,_,$(PLATFORM))
+VERSION          = 0.6.0
```
Comment (rakibulhossain) [inline Makefile]: remove?

**Review by rakibulhossain — CHANGES_REQUESTED** [review]
remove empty file role_scripts/15/standby/postgresql.conf

**Review by rakibulhossain — DISMISSED** [review]
LGTM!

**Review by AbdullahAlShaad — APPROVED** [review]
LGTM!

**Pull request merged**